website/content/en/docs/upgrading/v1-migration.md

```diff
@@ -112,27 +194,79 @@ kubectl patch customresourcedefinitions ec2nodeclasses.karpenter.k8s.aws -p "{\"
+
+   Alternatively, you can install the CRDs with the main chart and apply the following patches. However, we strongly recommend using the dedicated CRD chart.
+
+   ```bash
+   SERVICE_NAME="karpenter"
+   SERVICE_NAMESPACE="kube-system"
+   SERVICE_PORT="8443"
+   CRDS=("nodepools.karpenter.sh" "nodeclaims.karpenter.sh" "ec2nodeclasses.karpenter.k8s.aws")
+   for crd in ${CRDS[@]}; do
+       kubectl patch customresourcedefinitions ${crd} --patch-file=/dev/stdin <<-EOF
+   spec:
+     conversion:
+       webhook:
+         clientConfig:
+           service:
+             name: "${SERVICE_NAME}"
+             namespace: "${SERVICE_NAMESPACE}"
+             port: ${SERVICE_PORT}
+   EOF
+   done
+   ```
+   {{% /alert %}}
+
+   {{% alert title="Note" color="primary" %}}
+   Helm uses annotations on resources it provisions to track ownership.
+   Switching to the new chart may result in Helm failing to install the chart due to `invalid ownership metadata`.
+   If you encounter errors at this step, consult this [troubleshooting entry]({{<ref "../troubleshooting/#helm-error-when-installing-the-karpenter-crd-chart" >}}) to resolve.
+   {{% /alert %}}
+
+
+6. Validate that Karpenter is operating as expected on this latest patch release.
+   If you need to rollback after upgrading to `v1`, this is the version you will need to rollback to.
+
+   {{% alert title="Note" color="primary" %}}
+   The conversion webhooks must be able to communicate with the API server to operate correctly.
+   If you see errors related to the conversion webhooks, ensure that your security groups and network policies allow traffic between the webhooks and the API server.
+   {{% /alert %}}
+
+7. We're now ready to begin the upgrade to `v1`. Set the `KARPENTER_VERSION` environment variable to the latest `v1.0.x` release.
 
     ```bash
-    export KARPENTER_VERSION=1.0.3
+    export KARPENTER_VERSION="1.0.6"
     ```
 
-
-8. Update your existing policy using the following to the v1.0.3 controller policy:
+8. Attach the v1 policy to your existing NodeRole.
    Notable Changes to the IAM Policy include additional tag-scoping for the `eks:eks-cluster-name` tag for instances and instance profiles.
 
+   <!-- TODO: Pin ref to a specific commit once this commit has been merged -->
     ```bash
-    export TEMPOUT=$(mktemp)
-    curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/v"${KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml > ${TEMPOUT} \
-        && aws cloudformation deploy \
-        --stack-name "Karpenter-${CLUSTER_NAME}" \
-        --template-file "${TEMPOUT}" \
-        --capabilities CAPABILITY_NAMED_IAM \
-        --parameter-overrides "ClusterName=${CLUSTER_NAME}"
+    TEMPOUT=$(mktemp)
+    curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/website/content/en/docs/upgrading/v1-controller-policy.json > ${TEMPOUT}
```

Comment:
Do we know for sure there isn't more that we're missing from the cloudformation template?

Reply:
It's the only meaningful change for `v1`. There was an update to the resource policy for the interruption queue that we made across all versions that isn't captured here. I'm not sure how important it is to capture it, it isn't coupled to the version, but we can probably capture it by applying the proper cloudformation template in the final step rather than removing the `v1beta1` policy.

Reply:
I do still want to manually attach the policy at this step, rather than apply a CFN template with both policies, to make it clear what this step is doing for those who manage their IAM roles through something other than CFN (e.g. terraform).